jdisc_http_service/src/main/java/com/yahoo/jdisc/http/server/jetty/HealthCheckProxyHandler.java

```diff
@@ -124,21 +115,86 @@ protected void doStop() throws Exception {
         super.doStop();
     }
 
+    private static class ProxyRequestTask implements Runnable {
+
+        final AsyncContext asyncContext;
+        final ProxyTarget target;
+        final HttpServletResponse servletResponse;
+        final ServletOutputStream output;
+
+        ProxyRequestTask(AsyncContext asyncContext, ProxyTarget target, HttpServletResponse servletResponse, ServletOutputStream output) {
+            this.asyncContext = asyncContext;
+            this.target = target;
+            this.servletResponse = servletResponse;
+            this.output = output;
```

Comment:
Super-nitty: this field could just be a local variable in the `run` method.

Reply:
`getOuputStream()` may throw `IOException`, so I want to do it in the `handle` method.